### PR TITLE
LPS-157505 rename /admin/*jsp on knowledge base module

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
- LPS-157505 you can not subscribe to a template but you can comment. prevent npe
- LPS-157505 rename /admin/*jsp on knowledge base module

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/kb_template_comments.jsp [hunk 1]
```diff
@@ -62,10 +62,10 @@ boolean helpful = BeanParamUtil.getBoolean(kbComment, request, "helpful", true);
 							<c:if test="<%= kbComment != null %>">
 
 								<%
-								request.setAttribute("template_comment.jsp-kb_comment", kbComment);
+								request.setAttribute("kb_template_comment.jsp-kb_comment", kbComment);
```
Comment (AliciaGarciaGarcia) [inline modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/kb_template_comments.jsp:65]: @boton @ambrinchaudhary 

This is ok to change?

**Comment by AliciaGarciaGarcia** [comment]
ci:test:knowledge-base

File: modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/kb_template_comments.jsp (hunk 1 above)
Comment (boton) [reply modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/kb_template_comments.jsp:65]: LGTM. But not tested

Comment (ambrinchaudhary) [reply modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/kb_template_comments.jsp:65]: Just checked you also changed it here, so yes, I'm ok with the change ;-) 
Screenshot 2022-08-04 at 12 53 00

**Review by ambrinchaudhary — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:test:knowledge-base

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**